src/main/java/frc/robot/subsystems/Auto.java

```diff
@@ -283,8 +285,8 @@ private void Drive(int inches) {
             // establish setpoints for end of travel
             leftTarget = chassis.flep + revs;
             rightTarget = chassis.frep + revs;
-            chassis.setPositionMode();
-            chassis.setPositionTarget(leftTarget, rightTarget);
+
+            chassis.pathDrive(leftTarget, rightTarget, leftTarget, rightTarget);
```

Comment:
I think @pyjshtml re-did this the other day?

Reply:
the back wheels would have thrown errors and tried to drive at velocity 0

Reply:
I made fix for it

Reply:
I think we intended to put the back where in zero power/coast mode for this. In position mode, they will fight the front wheels at the end of long drives because they have different wheel diameters, unless we can compensate for that.

Reply:
I see the fix now, thanks